notebooks/Advanced_guide.ipynb

```diff
@@ -0,0 +1 @@
FVDcEF+idp+8to1GxqRWMURityAohFqfGkqcpZy++ul0L/8aiReAnrGJ++ZJPPvklIfTcun2XPMu5fv0GVV3z/Pkz6roEAmbYfIudEkZt8zFeRVogElkHCF1fbS0NY/f1uxrujROv0niSDbu57z1/8fDH/M7t3/r7Xqv/z42P5p9wcHCMPU44Or7BaLzP8+fPJORsN3jCJKi+Q1uNwWB1j+pctHd2KNWilN7mmViDd4J6eO/ErddbDBrXOGpd4fHk44JEW0mDJXoP4GmqmtVqQe+aK/3J3nm6zpMkGmMGq/roF8NgX77DaTECiyoU/bAghyGqweP7lrYpMTrl8PBUXIntC5TRzJdzXr54iTKKaBWLUpKDUhQFNklYrUuJSL/zlhyf7xlNJGm1jhlaQREVTfJg+SpulcDwbBbwIGanDNEQWikSY3j68EuePX6EAW7cuM73fvu3+e5vfx+04tMv7vPnP/2QDz74kMvLc0LkDQxDGxN5ENv3VXqoYGQxXa5WtA8b1uUqQu8lVVXtqM0ibqcVKvSS2NpErgQw7J2VVvSuZX82ZTySgMym6siKlCKfxrnlY8hdgVYdTV3xrHxM0ziOj07J8zFFnlDkKcELwdc5T1s51uuaxCYcHh5hkoTReEa5arj/xUNWq5Ihd2P3TP+68/51/+ZDIGghKW6QROe3iK0xWCteMdaIbbqgqp71asFPfvIjrIGuqnjnvfeZHhyTpwl1VXF6eo1/9s/+PZ49fcqDLx+ANuJhpTTaQFWXfPH5v9dTLwAAIABJREFUZ8z29kiSlOMbN7l+4wa3b9/h7OUlXdOJd08YCuMt8hwCBC9BrmdnZ2LG13Vbr6Bh7YstohDC9t9fPSVeWiNKadBSEGktEykEMUxcr0vhFxU5im57vsPV82mMJTF2Y1wZIlEwyzL29vaYTqckNuEsvj7NMrquiSG14PpA0zomXuH7QNs4et+LsVoq68FqtaKsqg0iWyQSu9A00XDTOQIZqIBV+goHrus66rpGKbEWSNIEqw1aQ98L94pEjqNtK6az8cZDajKZcHJyMvyVvheriLoRq4lZ35OPJGLDasNoXFBkhvF0hKwtehMRMZ3u0TQdSZKRpll0AU45ONgmYGttmIyn5FnBerGgD108zkDnZHOJCmjdy58G/tV//6/QGiazCcenR+wd7HF8csTJtWNmszHjPKEY5VgjjrptbOt4L6albdvhvUaRYG3BtZO7vPP2tzg+vMnFxYq/+slf8YuP/5x8pFDG4L3CWOH9eQPKBawuKMZi52+i2jUET93WtK5j/2CfJC/IxynLRcFqsaJc17jGiaHkKMMkNprlhRg9IpYlQ4vw6ywPvt5cziYo7dFGzNTK9SWPHn3OdDpm//AYYzSzvQNW6zIavDlAkniHvpoOWzfcrUfKANcqzE7R8uvGrqPjq0gKIIRCvTWUCiHwH/+L//TKze26jsViQe99lGlNqOomTg6P71r6VuLYh/ccHnB939O2tSwYsU2wuWF3jxMg8l9e3fENw4SB33NlOd4uNFrUK+BRepDjRpOpYBiPJxzt73FwcMi1azfZPzjh+OgElGX/wLC3f8zzl2c8f/mCy4uznTcWNYJRSgqQNCNxLp4DF0m5KrbhNGgLytN5ccl0g3FdkKV1kJVaaxlNRqKKajpwskMoV2vKck0xHmza42EYi3OB9boGDDZRKGVwTnx/lDYiAY7D+0DvJevJ9eLcq5SRVkslRNLed1RVxdHxMcenB8wXS4Le5/TaCeeXZ5Jq7WN7JLHxhjFUTcN6vbyS9JqkKXv7+6R5xsXlEtdLS1FkgmHTBt3N2trd6Q4EbLeZI8PnNijf8/LxY1bnF7x25y4//OE/4tu/8R16o/n8wQP+7Mc/4q//+uecvTzHKMisJuzcokEpcZiOPDCZLuIKKqRL4VTUdcWzp08w1goZN0ipogYlk4ptBK3QUbI7mDCKt420nro20NQJy4Xi80/vs1qVvP2Nt7hx8wZ936G0J7Gek5PTjWlU1zmKImM6K+hdS+dqiiJjMh6RZQXGpHQNPHn0CZfnc67dOOb1e2/Sdj0ff/w5i0UpxElrUF5JOOvfcXzVg3x77rb0Q+H4CtIgKWpdlIhrlPLgXWzdaBbzc378l38OzqG05dvfm5KOZlRti+t7vvX+t/n93/9D/pv/7r+lrkvyREjXwUvA2Pn5Sz76m1+Q5znJqODw4ICTkxMm4ymrxYo29JIIHtyOKaGOXDpRjVRVtSmIN5lAO+vHFWfcQUywc058iMnOmsgXEUJqnqYYbWidcNJCIHqbVFdQ7d2RJAmJsdGnJQYWxvb2dDzm9s2bjMcTtitPbM96Cdx7/PiJ+FXFLCr5HNI6z7OMpq4pyxWT2YzZRFKPFRbQuK4hKuGjt1OPTq527K1NyFI5P70PuM7RB+g6CUcUoryhLBuatuZw/3QTB6K1FB1XwyPkvKxXazrXk5dS6Bwd7nPt2glWB7zraJqaFy9fkiQFRTGh7xVZPsEHhesDxiaMYut2oA20Tbtpi1hrMCEI2VhLZpg4pXco3TOZ5Oztzbh+esrtOze5+9odbt+9zXg6QltF2zVUVUlbVXjvqDoxC+xiobv1ONNRuu1YLi5p6p48n5ImY/b2DnjjzTd5MX/ExeVTEqMwNgDD/SQ/a5OC8bjCJhqUUCG0UjRty2Kx5OjkiFR79lTBZJLRHM64PJ+zuBQurLGQJCoq4HzsyAzHpsREdSfv6tXxtUXLX/zzf/RrvtMCj3d+xWtf92u+cvwfP13yH/5nX+4QdLfjq1pDw59X4fjYv1YR1zFCZtoUN5HgS4iWwVr04/KA1NHtEXkQKw3a4PVV2fRQvAjLPPaJGR5Gniv39CA/3N0J/8pnQ6TT8esQNr8sKgekR4yKfekNmiQPzsVyhep7XNfTtp512TGdHWCTnDQtyLKCm3deY3xwxFnMWZE3MwRlopOvIstyxhNDUpYslmuccxI22TusSUmMQSuBX7u2FZt
```

Comment:
<p>This can be removed. It's the same as here above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lgvaz/mantisshrimp/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Removed</p>